src/test/java/org/jabref/gui/keyboard/KeyBindingsDialogViewModelTest.java

```diff
@@ -138,9 +140,9 @@ public void testSetAllKeyBindingsToDefault() {
     }
 
     @Test
-    public void testCloseEntryEditorCloseEntryKeybinding() {
+    void testCloseEntryEditorCloseEntryKeybinding() {
         KeyBindingViewModel viewModel = setKeyBindingViewModel(KeyBinding.CLOSE);
-        model.selectedKeyBindingProperty().set(viewModel);
+        model.selectedKeyBindingProperty().set(Optional.of(viewModel));
```

Comment:
Optionals should not be passed as parameter.

I agree that the code is more easy to call if the setter can be used in streams. Here, I see that the `viewModdel` is (nearly) always wrapped in `Optional.of` which makes it harder to read.

Reply:
Good point. That's a limitation of the current OptionalProperty. I've created a new issue https://github.com/tobiasdiez/EasyBind/issues/13 and will look at it in the coming days.